dags/sources/tasks/transform/transform_column.py

```diff
@@ -175,7 +175,7 @@ def clean_email(email: str | None, _) -> str:
 def clean_code_postal(cp: str | None, _) -> str:
     if pd.isna(cp) or not cp:
         return ""
-    # cast en str et ajout de 0 si le code postal est inférieur à 10000
+    cp = clean_number(cp)
     return f"0{cp}" if cp and len(str(cp)) == 4 else str(cp)
```

Comment:
remarque (non bloquante): peut-être qu'il serait intéressant de commenter ce genre de cas avec un lien vers l'issue initiale, ou donner un exemple qu'on cherche à résoudre avec ça ? (je l'ai déduit en lisant les tests, ce qui me semble aussi acceptable de dire "read unit test to understand logic behind this" ou un truc comme ca ?

Reply:
Plutôt dans la fonction clean_number alors :)